framework/src/st_things/things_stack/framework/things_resource.c

```diff
@@ -731,7 +731,7 @@ things_representation_s *things_create_representation_inst(void *rep_payload)
 	return rep;
 }
 
-things_resource_s *create_resource_inst_impl(void *requesthd, void *resourcehd, void *query, void *rep_payload)
+static things_resource_s *create_resource_inst_impl(OCRequestHandle requesthd, OCResourceHandle resourcehd, void *query, void *rep_payload)
 {
 	things_resource_s *res = (things_resource_s *) things_malloc(sizeof(things_resource_s));
```

Comment:
remove white space between ")" and "things_malloc"

Reply:
ok